src/EditorFeatures/VisualBasicTest/Completion/CompletionProviders/SymbolCompletionProviderTests.vb

```diff
@@ -6495,5 +6495,24 @@ End Class
             VerifyItemIsAbsent(text, "X")
         End Sub
 
+        <WorkItem(4900, "https://github.com/dotnet/roslyn/issues/4090")>
+        <Fact, Trait(Traits.Feature, Traits.Features.Completion)>
+        Public Sub ColorColorInstanceSharedRegression()
```

Comment:
I'm betting it fixes it off of the enum types but not off locals of an enum type.